src/Uno.Extensions.Authentication.MSAL/HostBuilderExtensions.cs

```diff
@@ -21,11 +21,48 @@ public static class HostBuilderExtensions
 	/// <returns>
 	/// The <see cref="IAuthenticationBuilder"/> that was passed in.
 	/// </returns>
+	[Obsolete("This method is obsolete. Please use the AddMsal overload that accepts a 'Window' parameter to specify the authentication window. The overload without 'Window' will be removed in a future release.", false)]
```

Comment:
We'll need to edit the current docs: https://github.com/unoplatform/uno.extensions/blob/78184f87453578f3fa6f9a9b1275ada1ef7b7fe6/doc/Learn/Authentication/HowTo-MsalAuthentication.md?plain=1#L51

Not sure if we have anywhere that we are documenting the upgrade experience from one version to another for Extensions. Similar to things like Toolkit: https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/material-migration.html

Though it's mostly for documenting upgrading to new major versions. Either way, let's make sure we do a pass where this should be added/called out in the documentation